src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -13,82 +13,95 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
-<span class="xHint">#t("clientid.hint")</span>
-: $configDoc.display("clientid", 'edit', $obj)
+<dl><dt><label for="${prefix}_clientid">#t("clientid")</label> ##
+    <span class="xHint">#t("clientid.hint")</span></dt>
+    <dd> #displayInput("clientid")</dd></dl>
 
-; <label for="${prefix}_secret">#t("secret")</label> ##
-<span class="xHint">#t("secret.hint")</span>
-: $configDoc.display("secret", 'edit', $obj)
+<dl><dt><label for="${prefix}_secret">#t("secret")</label> ##
+<span class="xHint">#t("secret.hint")</span></dt>
+    <dd> #displayInput("secret")</dd></dl>
 
 ## checkboxes for scope, needs JS
-; <label>#t("scope")</label> ##
-<span class="xHint">#t("scope.hint")</span>
-: <label title="#t('checkbox_mandatory')"> ##
+<dl><dt><label>#t("scope")</label> ##
+<span class="xHint">#t("scope.hint")</span></dt>
+<dd> <label title="#t('checkbox_mandatory')"> ##
 <input type="checkbox" name="scope_identity" disabled="true" checked>#t("scope_identity")</label> ##
 <label title="#t('checkbox_mandatory')"> ##
 <input type="checkbox" name="scope_email"    disabled="true" checked>#t("scope_email")</label> ##
 <label title="#t('checkbox_mandatory')"> ##
 <input type="checkbox" name="scope_drive"                           >#t("scope_drive")</label> ##
 <label title="#t('checkbox_notyetdone')"> ##
 <input type="checkbox" name="scope_avatar"                          >#t("scope_avatar")</label> ##
-$configDoc.display("scope", "hidden")
+    <input type="hidden" name="GoogleApps.GoogleAppsConfigClass_0_scope" value="${obj.getValue('scope')}"/></dd></dl>
 
-; <label for="${prefix}_appname">#t("appname")</label> ##
-<span class="xHint">#t("appname.hint")</span>
-: $configDoc.display("appname", 'edit', $obj)
+    <dl><dt><label for="${prefix}_appname">#t("appname")</label> ##
+<span class="xHint">#t("appname.hint")</span></dt>
+        <dd> #displayInput("appname")</dd></dl>
 
 </fieldset>
 
 #if($services.googleAppsService.missesAuthenticationConfig())
-    {{warning}}$services.localization.render('googleapps.error.authConfigNotFound')
-    $services.localization.render('googleapps.error.authConfigShould',['[[','>>https://store.xwiki.com/xwiki/bin/view/Extension/GoogleAppsIntegration#installation]]']):
-
-    {{code}}xwiki.authentication.authclass=com.xpn.xwiki.user.impl.xwiki.GroovyAuthServiceImpl
-    xwiki.authentication.groovy.pagename=xwiki:GoogleApps.AuthService{{/code}}
-    {{/warning}}
-
+    <div class="box warningmessage">
+        <p>$services.localization.render('googleapps.error.authConfigNotFound')<br>
+            #tp('googleapps.error.authConfigShould',['[[','>>https://store.xwiki.com/xwiki/bin/view/Extension/GoogleAppsIntegration#installation]]'])</p>
+        <div class="box"><div class="code"><span style="border: 1px solid #FF0000; ">xwiki.authentication.authclass=com.xpn.xwiki.user.impl.xwiki.GroovyAuthServiceImpl</span><br><span style="border: 1px solid #FF0000; ">xwiki.authentication.groovy.pagename=xwiki:GoogleApps.AuthService</span>
```

Comment:
The inline styling is not a good practice and we should use a `SSX` object to write `css` or `less` code.
Also we should care to not exceed the 120 chars per line. Some indentation, extraction in variables, should help.

Reply:
I had taken the copy of the produced html.
I now simplified it.